release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1472,7 +1482,7 @@ This release removes the Prometheus Adapter, which was a Technology Preview feat
 
 *ImageStreams*
 
-
+* Previously, performance was sometimes slow when importing multiple images from a stream. With this release, the number of concurrent requests to the image registry is increased from five to 50, and performance is improved. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954715[*BZ#1954715*])
```

Comment:
s/50, and performance is improved/50. As a result performance is improved   or resulting in improved performance

(something to make this seem like it is not two changes)